features/ubuntu_upgrade.feature

```diff
@@ -162,3 +162,44 @@ Feature: Upgrade between releases when uaclient is attached
         | release | next_release | fips-service  | fips-name    | source-file         |
         | xenial  | bionic       | fips          | FIPS         | ubuntu-fips         |
         | xenial  | bionic       | fips-updates  | FIPS Updates | ubuntu-fips-updates |
+
+    @slow
+    @series.bionic
+    @uses.config.machine_type.lxd.container
+    @upgrade
+    Scenario Outline: Attached upgrade with cis enabled across LTS releases
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token_staging` with sudo
+        And I run `ua enable cis` with sudo
+        # update-manager-core requires ua < 28. Our tests that build the package will
+        # generate ua with version 28. We are removing that package here to make sure
+        # do-release-upgrade will be able to run
+        And I run `apt remove update-manager-core -y` with sudo
```

Comment:
No change needed for this PR, but this discovery is concerning. We may need to sync with the update-manager team to see why this is so, and to talk about versioning/backwards-compat going forward.

Reply:
@orndorffgrant worth a card IMHO

Reply:
https://warthogs.atlassian.net/browse/SC-703